ci/build/build-vscode.sh

```diff
@@ -47,6 +47,7 @@ main() {
   # as it is a submodule.
   export VSCODE_DISTRO_COMMIT
   VSCODE_DISTRO_COMMIT=$(git rev-parse HEAD)
+  VERSION=$(jq -r .version ../../package.json)
```

Comment:
We still need to delete this line otherwise the version we set in the workflow gets overridden!